Main.js

```diff
@@ -86,7 +86,72 @@ function decrease_zoom() {
 }
 
 function reset_zoom () {
-	change_zoom(60.0 / window.CURRENT_SCENE_DATA.hpps);
+	console.group("reset_zoom")

```

Comment:
This weird indentation isn't visible in vscode so assuming github is doing something stupid?!